privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -14,10 +14,11 @@ <h2>{% ftlmsg 'multi-part-onboarding-premium-welcome-subheadline' %}</h2>
 
                 <div class="c-premium-onboarding-content">
                     <div class="c-premium-onboarding-content-image">
-                        <img src="{% static 'images/dashboard-onboarding/woman-couch.svg' %}" alt="Woman sitting on a couch">
+                        <img src="{% static 'images/dashboard-onboarding/woman-couch.svg' %}" alt="">
                     </div>
                     <div class="c-premium-onboarding-content-description">
-                        <p><strong>{% ftlmsg 'multi-part-onboarding-premium-welcome-title' %}</strong> {% ftlmsg 'onboarding-premium-control-description' %}</p>
+                        <p class="c-premium-onboarding-detail">{% ftlmsg 'onboarding-premium-title-detail' %}</p>
+                        <p><strong>{% ftlmsg 'multi-part-onboarding-premium-welcome-title' %}</strong> {% ftlmsg 'multi-part-onboarding-premium-welcome-description' %}</p>
```

Comment:
I just realised that `multi-part-onboarding-premium-welcome-description` says

> On desktop, you can use the Relay add-on to create them on the go.

So maybe here, too, we need to display different content depending on the screen size?

Reply:
Ohhh cool, I had no idea I could use admin for that. Solves so many testing hurdles! Lol.